package.json

```
@@ -128,7 +128,7 @@
         "@box/cldr-data": "^34.2.0",
         "@box/frontend": "^10.0.0",
         "@box/languages": "^1.0.0",
-        "@box/metadata-editor": "^0.46.1",
+        "@box/metadata-editor": "^0.50.4",
```

```suggestion
        "@box/metadata-editor": "^0.50.5",
```
to avoid conflicts, we should be using the same version

---

fixed